parlai/core/teachers.py

```diff
@@ -163,6 +163,7 @@ class Teacher(Agent):
     def add_cmdline_args(
         cls, parser: ParlaiParser, partial_opt: Optional[Opt] = None
     ) -> ParlaiParser:
+        parser.add_argument('--seed', default=None)
```

Comment:
can we change this to `--teacher-seed`? to differentiate from the `--seed` already in train_model.py?